projects/ad713x_fmcz/src/parameters.h

```diff
@@ -52,7 +52,7 @@
 #define GPIO_DEVICE_ID			XPAR_PS7_GPIO_0_DEVICE_ID
 #define AD7134_DMA_BASEADDR		XPAR_AXI_AD7134_DMA_BASEADDR
 #define AD7134_SPI_ENGINE_BASEADDR	XPAR_DUAL_AD7134_DUAL_AD7134_AXI_REGMAP_BASEADDR
-#define AD713x_SPI_ENG_REF_CLK_FREQ_HZ	96000000
+#define AD713x_SPI_ENG_REF_CLK_FREQ_HZ	100000000
```

Comment:
Can you please provide some context here ? There is this 96->100 MHz ref clock change and also SPI mode 3 to 0 change - do these mean that the project was not at all functional before ? Or did the FMCZ board change revisions and requires these changes ?